paf-mvp-operator-express/src/operator-node.ts

```diff
@@ -56,14 +59,20 @@ export enum Permission {
   WRITE = 'WRITE',
 }
 
-export type AllowedDomains = { [domain: string]: Permission[] };
+export type AllowedHosts = { [host: string]: Permission[] };
+
+export interface OperatorConfig {
```

Comment:
WDYT of renaming this to `OperatorNodeConfig`, given that there is `ClientNodeConfig` for the `ClientNode` class, and that it's for the `OperatorNode` class?

Reply:
thanks, done